### ISSUE TITLE
Crash with latest support libraries 

### ISSUE THREAD

**Issue by rubengees** [consp1racy/android-support-preference#90]
With the latest support library version (`27.1.0`) the drop down lists crash due to `ListViewCompat` being removed now.

A full stacktrace:

```java
java.lang.NoClassDefFoundError: Failed resolution of: Landroid/support/v7/widget/ListViewCompat;
        at net.xpece.android.support.preference.ListPreference.showAsPopup(ListPreference.java:175)
        at net.xpece.android.support.preference.ListPreference.performClick(ListPreference.java:130)
        at android.support.v7.preference.Preference$1.onClick(Preference.java:170)
        at android.view.View.performClick(View.java:6303)
        at android.view.View$PerformClick.run(View.java:24828)
        at android.os.Handler.handleCallback(Handler.java:789)
        at android.os.Handler.dispatchMessage(Handler.java:98)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6809)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```

**Comment by consp1racy** [comment]
There were changes even to preference-v7 in 27.1.0, I'll have to revise.

**Comment by consp1racy** [comment]
Fix just released in v2.0.1.
